src/styles/blocks/product-button.scss

```diff
@@ -0,0 +1,24 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
Mentors say the best practice is to place this property 
 
>  `box-sizing: border-box;`

 to `* {
} `selector)